frontend/src/components/Play.tsx

```diff
@@ -0,0 +1,49 @@
+import {ChangeEvent, FormEvent, useState} from "react";
+import {PromptCreate} from "../types/PromptCreate";
+import {useParams} from "react-router-dom";
+import axios from "axios";
+
+export default function Play() {
+    const params = useParams()
+    const gameId: string | undefined = params.gameId
+    const [prompt, setPrompt] = useState<string>("")
+
+    const submitPrompt = async (promptToSubmit: PromptCreate) => {
+        try {
+            await axios.post(`/api/play/${gameId}/prompt`, promptToSubmit)
```

Comment:
Warum nutzt du hier await? Da kannst du ja eigentlich auch die klassische .catch syntax nutzen

Reply:
Diese Funktion ist leider noch "work in progress". Mit der Response geht es später weiter. Aktuell ignorier ich sie. Sonst wäre das PR noch riesiger und später gekommen.